packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -40,6 +40,8 @@ import { AuctionState } from './util.js';
  * @import {PriceAuthority, PriceDescription, PriceQuote, PriceQuoteValue, PriceQuery,} from '@agoric/zoe/tools/types.js';
  */
 
+const BASIS_POINTS = 10_000n;
```

Comment:
There is defined 14 times. I maintain it's worth keeping in a common constants, but that's not useful unless it's also imported. Regardless I agree it's not part of a `utils` module. So no objection to this change and it can be consolidated separately (at which point the 14 times will become 1).